### PR TITLE
Codeword Update Logfix 2.0

### PR DESCRIPTION
**Pull request by Kierany9**
If I've done this wrong I'm going to flip shit

(i apparently dont know how to update prs)

### CODE REVIEW

**Comment by Crystalwarrior** [comment]
stop creating seperate PR's, they track the branch you comitted as a PR. Just commit to the branch and done.

File: html/changelogs/Kierany9 - CWords.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+
```
Comment (Crystalwarrior) [inline html/changelogs/Kierany9 - CWords.yml]: CAN YOU LEARN TO EDIT TEXT FILES JESUS FUCKING CHRIST

**Comment by Kierany9** [comment]
It appears that in my anger...I killed the branch

**Comment by doobiedoo23** [comment]
Bugfix, merging

**Pull request merged**